README.md

```diff
@@ -1,12 +1,12 @@
 <a name="readme-top"></a>
-<div align="center">
+<!-- <div align="center">
 
   <img src="/murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
   <h3><b>Microverse</b></h3>
 
-</div>
+</div> -->
 <div align="left">
   <h1>The Bookstore CMS</h1>
 
```

Comment:
- [**Optional**] @habasm your README.md file contains two project titles. Below and above table of content.
![Screenshot from 2023-02-28 09-25-50](https://user-images.githubusercontent.com/63087491/221796295-a07fc906-a5f3-4fad-9cf3-ec22559ef376.png)